Assets/MixedRealityToolkit/Definitions/BaseMixedRealityProfile.cs

```diff
@@ -1,10 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System;
 using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit
 {
+    [Serializable]
```

Comment:
Oh! I thought `ScriptableObject`s were serializable by default.
What's this new attribute enable?

Reply:
I think they are but the compiler was complaining about it as a warning. Figured safe to add

Reply:
Cool works for me!